_data/nav.yml

```diff
@@ -727,9 +727,9 @@
     - url: "#Overview"
     - url: "#TinyMCE 5.3 new features and enhancements"
     - url: "#Accompanying Premium Plugin changes"
+    - url: "#Accompanying premium self-hosted server-side component changes"
```

Comment:
Why is `Premium` capitalised above but not here?

Reply:
ln 729: Noun
In 730: not a Noun

Reply:
to expand, opting to use sentence case, rather than title case, so only nouns and first word get a leading capitalised letter